### ISSUE TITLE
Error when running model.fit after upgrade tensorflow.net to v0.150

### ISSUE THREAD

**Issue by kojack14** [SciSharp/TensorFlow.NET#1238]
### Description

I am getting this error : Unable to find an entry point named 'TF_GetHandleShapeAndType' in DLL 'tensorflow'. My code :
for (int j = 0; j < trainingParameters.loopQtd; j++)
{

    var samples = PrepareDataTraining(trainingParameters, ref Id, j, 1, scheduleModel.id_aa,trainingParameters.modeltype);
    var test = TestDataNDArray(samples, trainingParameters.testRatio);
    test = test.reshape(new Shape(1, (int)test.shape[0], (int)test.shape[1]));
    test = Transpose(test, 1, (int)samples.shape[2], (int)samples.shape[1]);
    var history = trainingParameters.model.fit(
                samples,
                samples,
                trainingParameters.bacthsize,
                epochs: 1,
                verbose: 1,
                validation_data: (test, test),
                use_multiprocessing: false
                );
    auxLoss.add(history.history["loss"]);
    auxvalLoss.add(history.history["val_loss"]);
    auxerroVal.add(history.history["val_mean_absolute_error"]);
    auxerroTrain.add(history.history["mean_absolute_error"]);
}
